tests/epics/motion/test_motor.py

```diff
@@ -59,7 +59,7 @@ async def test_motor_moving_well(sim_motor: motor.Motor) -> None:
         target=0.55,
         unit="mm",
         precision=3,
-        time_elapsed=pytest.approx(0.1, abs=0.05),
+        time_elapsed=pytest.approx(0.1, abs=0.2),
```

Comment:
``` suggestion
        time_elapsed=pytest.approx(0.1, abs=0.05),
```
The tolerance should never be more than the approximate value in my opinion - this should be taking a little more or less that 100ms every single time. With this change, any value `0 <= x <= 0.3` would be allowed...

We should work out why this takes ~ 0.2 seconds on that one CI build instead of ~ 0.1. This doesn't seem to happen on main.